osconfig_tests/compute/instance.go

```diff
@@ -89,6 +92,46 @@ func (i *Instance) WaitForSerialOutput(match string, port int64, interval, timeo
 	}
 }
 
+// WaitForGuestAttribute waits to a guest attribute to appear.
+func (i *Instance) WaitForGuestAttribute(queryPath, variableKey string, interval, timeout time.Duration) (string, error) {
```

Comment:
Since `interval` and `timeout` are always the same, should they be consts instead?

Reply:
This function is meant to be generic, the callers are all for osconfig_tests/install_done right now but in different packages so there isn't really a good place to put a constant